src/Microsoft.Data.Analysis/PrimitiveDataFrameColumn.cs

```diff
@@ -323,13 +324,24 @@ public override DataFrame ValueCounts()
             return new DataFrame(new List<DataFrameColumn> { keys, counts });
         }
 
+        /// <inheritdoc/>
         public override bool HasDescription() => IsNumericColumn();
 
+        /// <summary>
+        /// Prints a preview of the column contents as a formatted string.
```

Comment:
(nit) it doesn't really "print" anything. It returns a string.

Reply:
Yup. Fixed.